osf/models/preprint_provider.py

```diff
@@ -20,16 +30,25 @@ class PreprintProvider(ObjectIDMixin, BaseModel):
     banner_name = models.CharField(null=True, blank=True, max_length=128)  # max length on prod: 19
     domain = models.URLField(blank=True, default='', max_length=200)
     domain_redirect_enabled = models.BooleanField(default=False)
-    external_url = models.URLField(null=True, blank=True, max_length=200)  # max length on prod: 25
-    email_contact = models.CharField(null=True, blank=True, max_length=200)  # max length on prod: 23
-    email_support = models.CharField(null=True, blank=True, max_length=200)  # max length on prod: 23
     example = models.CharField(null=True, blank=True, max_length=20)  # max length on prod: 5
     access_token = EncryptedTextField(null=True, blank=True)
     advisory_board = models.TextField(null=True, blank=True)
+
+    emails = models.ManyToManyField(Email, blank=True)
+    # TODO: Remove email fields in favor of emails
+    email_contact = models.CharField(null=True, blank=True, max_length=200)  # max length on prod: 23
```

Comment:
Sorry, looked like you were adding it. During the migration all fields had to be nullable because of the way the migrations worked and the fact that our data was, previously, inconsistent. There was supposed to be a story to go back through and fix them all. Changing it and it's 8 friends is probably out of scope for your story. I'll bug @sloria to see if that story exists.